lib/classes/Variables.js

```diff
@@ -96,7 +97,9 @@ class Variables {
         } else {
           singleValueToPopulate = this.getValueFromSource(variableString)
             .then(valueToPopulate => {
-              if (typeof valueToPopulate === 'object') {
+              if (typeof valueToPopulate === 'object'
```

Comment:
populateObject doesn't appear to handle arrays (which have typeof === 'object')
could this be a potential risk?